conformance/zip.sh

```diff
@@ -18,15 +19,16 @@ do
   failure='<failure message="'
   close_failure="</failure>"
 
-  if [ $testline = "---" ] && [ $extendline != '----' ] # This checks if the line is a test line (starts with --- but not ----)
+  # This checks if the line is a test line (starts with --- but not ----)
+  if [ $testline = "---" ] && [ $extendline != '----' ]
   then
     status=$(echo $line | cut -c4-8) # The first four characters after --- are the pass/fail status of the test)
     name=$(echo $line | tail -c +11 | head -c -9) # The next string excluding the time is the name of the test
     echo "$testcase$name$end" >> artifacts/junit_01.xml
     if [ $status = "FAIL" ]
     then
       read line1
-      message=$(echo "$line1" | sed 's/</ /g; s/>/ /g; s/"//g') # $message is the failure message excluding <, >, ", which cause the xml file to fail
+      message=$(echo "$line1" | sed 's/&lt;/ /g; s/&gt;/ /g; s/&quot;/g') # $message is the failure message excluding <, >, ", which cause the xml file to fail
```

Comment:
What I meant was, change `<` to `&lt;`, etc., so it's displayed correctly.  Change the comment to something like "Quote the characters that XML might interpret.".  Consider quoting `&` as `&amp;` too.  Check an XML reference to see if this is sufficient.